lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -42,18 +42,39 @@ name:
   get_command: "show vlan brief"
   get_value: '/^%d\s+(\S+)\s/'
   set_context: ["vlan %d"]
-  set_value: "%s name %s"
+  set_value: "%s name %s ; end"
+
+private_vlan_association:
+  _exclude: [N8k]
+  kind: string
+  multiple: true
+  get_command: "show vlan private-vlan"
+  get_value: '/^<id>\s+(\d+)/'
+  set_context: ['vlan <vlan>']
+  # Need end to push the config Without the config wont take effect
+  set_value: "<state> private-vlan association <operation> <vlans> ; end"
+  default_value: "default"
```

Comment:
Done. I have created 3 properties since I did know how handle the "no" case. Let me know if it's ok @mikewiebe